### PR TITLE
CLUE Dice Roller

### PR DESCRIPTION
**Pull request by dastels**

### CODE REVIEW

File: CLUE_Dice_Roller/clue_dice_roller.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+button_a = Debouncer(lambda: clue.button_a)
+button_b = Debouncer(lambda: clue.button_b)
+
+
+# Set up display
+
+display = board.DISPLAY
+selection_font = bitmap_font.load_font('/Helvetica-Bold-36.bdf')
+selection_font.load_glyphs(b'0123456789')
```
Comment (ladyada) [inline CLUE_Dice_Roller/clue_dice_roller.py]: add xDd to preloaded glyphs

File: CLUE_Dice_Roller/clue_dice_roller.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+selection_color = 0x0000FF
+
+roll_font = bitmap_font.load_font('/Anton-Regular-104.bdf')
+roll_font.load_glyphs(b'0123456789')
```
Comment (ladyada) [inline CLUE_Dice_Roller/clue_dice_roller.py]: ditto

**Comment by dastels** [comment]
There's more commits than should be here

File: M4_Eyes/user.cpp [hunk 3]
```diff
@@ -1,5 +1,7 @@
 #if 1 // Change to 0 to disable this code (must enable ONE user*.cpp only!)
 
+#include "globals.h"
```
Comment (ladyada) [inline M4_Eyes/user.cpp]: remove this file

**Pull request closed without merging**